src/components/business/business.ts

```diff
@@ -0,0 +1,57 @@
+import { defineComponent, ref } from 'vue'
+import ChipItem from '../chip/Chip.vue'
+import Profile from '../profile/Profile.vue'
+
+export interface TeamData {
+  name: string
+  jon: string
+  image: string
+  tag: string
+}
+
```

Comment:
Pas d'interfaces, utilise des types. Et un fichier par type, le tout dans un dossier src/types

```suggestion
type TeamData = {
  name: string
  jon: string
  image: string
  tag: string
}

export default TeamData

```